src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -117,16 +120,13 @@ public function getFunctions(): array
     {
         $functions = [
             new TwigFunction('is_granted', $this->isGranted(...)),
+            new TwigFunction('is_granted_for_user', $this->isGrantedForUser(...)),
             new TwigFunction('impersonation_exit_url', $this->getImpersonateExitUrl(...)),
             new TwigFunction('impersonation_exit_path', $this->getImpersonateExitPath(...)),
             new TwigFunction('impersonation_url', $this->getImpersonateUrl(...)),
             new TwigFunction('impersonation_path', $this->getImpersonatePath(...)),
         ];
 
-        if ($this->userSecurityChecker) {
```

Comment:
shouldn't we keep this with a check `$this->securityChecker instanceof UserAuthorizationCheckerInterface` (allowing to fail at compile time instead of failing at runtime when calling the function, which would also be compatible with `{% guard %}`)